### DISCUSSION TITLE
Archived Discussions

### DISCUSSION THREAD

**Discussion by manngo** [atom/atom#22965] [Q&A]
I’m a little confused about the move from `discussion.atom.io` to GitHub. In particular there is a discussion that I wanted to look at again, but I get the __Not Secure__ warning, and the old discussion is inaccessible.

Is there a way to look at an archived copy of that discussion, or is it lost?

**Comment by nazimboudeffa** [comment]
I just came back after testing VSCode for a while to look at some old threads and I don't find a way of looking into the old forum 

Maybe there is a chance with ARCHIVE.ORG [https://web.archive.org/web/20140923170629/https://discuss.atom.io/](https://web.archive.org/web/20140923170629/https://discuss.atom.io/)

Also is Atom's Slack up or we move to Discord ?

**Reply by idleberg** [reply]
I've submitted a couple of threads that were relevant to me to Wayback Machine, but unfortunately, that does not trigger crawling the rest of the forum. But the main problem with backups on Wayback Machine is that people don't usually stumble across them through a web search – you have to know what you're looking for (or use some browser extension forwards to it).
